source/test/TestApp/PlayFabApiTest.cpp

```diff
@@ -16,11 +16,48 @@
 #include "PlayFabApiTest.h"
 #include "TestContext.h"
 
+#include <random>
+#include <sstream>
+
 using namespace PlayFab;
 using namespace ClientModels;
 using namespace AuthenticationModels;
 using namespace DataModels;
 
+namespace uuid {
+    static std::random_device              rd;
+    static std::mt19937                    gen(rd());
+    static std::uniform_int_distribution<> dis(0, 15);
+    static std::uniform_int_distribution<> dis2(8, 11);
+
+    std::string generate_uuid_v4() {
```

Comment:
Generating a random guid would be a great general utility, not just a random method in a test.